packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -3,16 +3,31 @@ import * as React from 'react';
 import * as ReactTestUtils from 'react-dom/test-utils';
 import * as renderer from 'react-test-renderer';
 import { SpinButton } from './SpinButton';
-import { KeyCodes } from '../../Utilities';
+import { KeyCodes, resetIds } from '../../Utilities';
 import { mockEvent, renderIntoDocument } from '../../common/testUtilities';
 
 describe('SpinButton', () => {
+  beforeEach(() => {
+    resetIds();
+  });
+
   it('renders SpinButton correctly', () => {
     const component = renderer.create(<SpinButton label="label" />);
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders SpinButton correctly and value should change accordingly even when no props are defined', () => {
```

Comment:
Bit more to the point :)
```suggestion
  it('allows value updates when no props are defined', () => {
```

Reply:
When writing "it" descriptions, read it by appending it to the thing you're describing. In this case, here's why it's awkward:

`SpinButton renders SpinButton correctly and value should change...`

And why @ecraig12345 suggestion reads better:

`SpinButton allows value updates when no props are defined`